### PR TITLE
Taking SAS token timeout as input

### PR DESCRIPTION
**Pull request by 20shivangi**

### CODE REVIEW

File: Tasks/AzureFileCopyV3/task.json [hunk 1]
```diff
@@ -226,6 +226,13 @@
             "defaultValue": "",
             "groupName": "output",
             "helpMarkDown": "Provide a name for the variable for the storage container SAS Token used to access the files copied to with this task.  Valid only when the selected destination is Azure Blob."
+        },
+        {
+            "name": "sasTokenTimeOutInMinutes",
+            "type": "string",
+            "label": "SAS Token Expiration Period In Minutes",
+            "required": false,
+            "helpMarkDown": "Enter the time in minutes after which SAS token expire."
```
Comment (niadak) [inline Tasks/AzureFileCopyV3/task.json]: Enter the time in minutes after which SAS token will expire.

Comment (20shivangi) [reply Tasks/AzureFileCopyV3/task.json]: It has been done

File: Tasks/AzureFileCopyV3/task.json [hunk 2]
```diff
@@ -226,6 +226,13 @@
             "defaultValue": "",
             "groupName": "output",
             "helpMarkDown": "Provide a name for the variable for the storage container SAS Token used to access the files copied to with this task.  Valid only when the selected destination is Azure Blob."
+        },
+        {
+            "name": "sasTokenTimeOutInMinutes",
+            "type": "string",
+            "label": "SAS Token Expiration Period In Minutes",
+            "required": false,
```
[the remark this answers is not in the document]
Comment (20shivangi) [reply Tasks/AzureFileCopyV3/task.json]: It has been done

File: Tasks/AzureFileCopyV3/task.json [hunk 3]
```diff
@@ -226,6 +226,13 @@
             "defaultValue": "",
             "groupName": "output",
             "helpMarkDown": "Provide a name for the variable for the storage container SAS Token used to access the files copied to with this task.  Valid only when the selected destination is Azure Blob."
+        },
+        {
+            "name": "sasTokenTimeOutInMinutes",
+            "type": "string",
+            "label": "SAS Token Expiration Period In Minutes",
```
[the remark this answers is not in the document]
Comment (20shivangi) [reply Tasks/AzureFileCopyV3/task.json]: It has been done

File: Tasks/AzureFileCopyV3/AzureFileCopy.ps1 [hunk 4]
```diff
@@ -24,14 +24,23 @@ $skipCACheck = Get-VstsInput -Name SkipCACheck -AsBool
 $enableCopyPrerequisites = Get-VstsInput -Name EnableCopyPrerequisites -AsBool
 $outputStorageContainerSasToken = Get-VstsInput -Name OutputStorageContainerSasToken
 $outputStorageURI = Get-VstsInput -Name OutputStorageUri
+$sasTokenTimeOutInMinutes = Get-VstsInput -Name SasTokenTimeOutInMinutes
```
Comment (niadak) [inline Tasks/AzureFileCopyV3/AzureFileCopy.ps1]: Make this call inside if clause

**Pull request merged**